index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,900;1,100;1,300&display=swap"
+      rel="stylesheet"
+    />
+    <title>Gideon's Portfolio</title>
+  </head>
+
+  <body>
+    <header class="template5-main-mobile">
+      <div class="app-bar">
+        <div class="time"><img src="Time.png" alt="time" /></div>
+        <h3 class="name">Gidiz Logo</h3>
+      </div>
+      <div class="top-circe">
+        <div class="ellipse1"></div>
+        <div class="ellipse2"></div>
+      </div>
```

Comment:
- [ ] Good job on adding the header but please add hamburger menu because it is not showing. 💪 
Here you have a screenshot of how I am seeing it:

![image](https://user-images.githubusercontent.com/74506933/153674401-fc3ec8ee-717c-45b0-85f3-58a141e61167.png)